.github/workflows/ci.yml

```diff
@@ -56,6 +56,7 @@ jobs:
 
       - name: Run ruby tests (hard-mode with GC.stress)
         run: bundle exec rake spec
+        if: matrix.os != 'macos-latest' # these specs are aborting on CI with no logs, maybe OOM killer?
```

Comment:
#156 is why it's aborting on macOS. Skipping meanwhile is fine by me, but I really want to fix that.

Reply:
Good call. We'll re-enable after the fix is in.